install_config/persistent_storage/topics/glusterfs_storage_sizing.adoc

```diff
@@ -0,0 +1,25 @@
+Every GlusterFS cluster must be sized based on the needs of the anticipated
+applications that will use its storage. Some additional things to consider are:
+
+- For each {gluster-native} or {gluster-external} cluster the default behavior
+is to create GlusterFS volumes with three-way replication. As such, the total
+storage to plan for should be the desired capacity times three.
+  ** As an example, each heketi instance creates a `heketidbstorage` volume
+  that is 2GB in size, requiring a total of 6GB of raw storage across three
+  nodes in the storage cluster. This capacity is always required and should
+  be taken into consideration for sizing calculations.
+  ** Applications like an integrated OpenShift Container Registry share a
+  single GlusterFS volume across multiple instances of the application.
+- gluster-block volumes require the presence of a GlusterFS block-hosting
+volume with enough capacity to hold the full size of the desired block volume's
```

Comment:
s/hold the full size of the desired block volume's/hold the full size for all of the desired block volumes

Reply:
I don't quite agree with the proposed change, but I see the confusion. Will reword.